app/assets/stylesheets/hoge/fuga.css.scss

```diff
@@ -0,0 +1,40 @@
+.cat{}
```

Comment:
__[scss_lint]__
* [SpaceBeforeBrace] Opening curly brace `{` should be preceded by one space
* [EmptyRule] Empty rule

<!-- Comment by [SideCI](https://sideci.com) -->